game/src/main_menu/mainmenu.rs

```diff
@@ -30,30 +30,32 @@ impl Default for MainMenu {
     fn default() -> Self {
         let (sender, receiver) = channel();
 
-        let start_button = Button {
-            text: "Start".to_string(),
-            img: None,
-            message: Start,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 180.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
-        let exit_button = Button {
-            text: "Exit".to_string(),
-            img: None,
-            message: Exit,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 420.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
-        let new_game_button = Button {
-            text: "New Game".to_string(),
-            img: None,
-            message: NewGame,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 300.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
+        let start_button = Button::new(
+            "Start".to_string(),
+            None,
+            Start,
+            sender.clone(),
+            graphics::Rect::new(1322., 350., 350.0, 120.0),
+            Color::from_rgba(195, 195, 195, 255),
+        );
+
+        let new_game_button = Button::new(
+            "New Game".to_string(),
+            None,
+            NewGame,
+            sender.clone(),
+            graphics::Rect::new(1322., 490., 350.0, 120.0),
```

Comment:
Vielleicht sollten wir mal ne Struct machen in der wir einfach Konstant Farben reinmachen, die wir dann überall nutzen... So ist das wirklich tedious und schlecht zu lesen, aber ist nicht im Scope dieses Commits

Reply:
Good call. Kann daraus gleich mal nen neuen Issue machen